### ISSUE TITLE
Question of pretrained model

### ISSUE THREAD

**Issue by yxldd** [Janspiry/Image-Super-Resolution-via-Iterative-Refinement#138]
What's the different between the _opt and _gen pretrained models?

**Comment by Jixuan-Fan** [comment]
Hi, @yxldd 
`_opt` is like .`state` file which contains training parameters like epoch / iter / scheduler / optimizer etc.
while `_gen` is like `.pth` file which contains model parameters.
You could find corresponding code here:
https://github.com/Janspiry/Image-Super-Resolution-via-Iterative-Refinement/blob/01d27a7cbfa8502be1d8dbd4ee02fcbd5e44389d/model/model.py#L129C1-L141C40
